### ISSUE TITLE
[Ability][Archmage] Cleansing Light (2)

### ISSUE THREAD

**Issue by Aluviya** [WarEmu/WarBugs#14490]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
Make sure you've checked the wiki before you post something ToK related:
https://wiki.returnofreckoning.com/

**Expected behavior and actual behavior:**
I already had opened an issue about this ability: https://github.com/WarEmu/WarBugs/issues/14421

There is another additional problem with this skill:
![Aluvya_031](https://user-images.githubusercontent.com/17249318/60354308-3b5f7300-99cc-11e9-9029-6bc9113625af.jpg)
If the tactic ![Aluvya_029](https://user-images.githubusercontent.com/17249318/60354212-07844d80-99cc-11e9-8143-f0f3919f6d32.jpg) **and** ![Aluvya_030](https://user-images.githubusercontent.com/17249318/60354213-081ce400-99cc-11e9-8088-2ac2e4ebc485.jpg) are used at the same time, while beeing flagged for rvr,  the hot from the tactics and the shield are **only** applied to the secondary target. In PvE every target in group that is cleansed gets a **hot** **and** a **shield**. If Apotheosis is slotted without  Isha's Encouragement the shield is applied to all party-members that are cleansed - regardless of beeing flagged or not. With having 2 Tactics invested this skill should apply a shield and a hot on all targets that are cleansed.

Btw: This is just another skill that seems to work different while beeing flagged or unflagged with the Archmage.

Same is for  ![Aluvya_032](https://user-images.githubusercontent.com/17249318/60354909-9e9dd500-99cd-11e9-9356-420c45e2bd0a.jpg)- which simply doesn't work at all if your target is not flagged.

**Comment by grunbag** [comment]
so the skill turn suddently aoe in Pve?

**Comment by Aluviya** [comment]
Let me try to explain again:
PvP flagged:
(1) If you use Cleansing Light **with** **Apotheosis**:
- Your target and all other targets are cleansed **and** all cleansed targets gain a **shield** (which is from the skill itself, check the image; skill removes a hex or ailment **and** gives a **shield** on that ally)

(2) If you use Cleansing Light **with** **Apotheosis** **and** **Isha's Encouragement**:
- Your target and all other targets are cleansed, but **only** the secondary target gets a **shield** and the **hot** (Ishas Encouragement), all other targets that were cleansed recieve **no shield nor HoT**

PvE flagged:
(3) If you use Cleansing Light **with** **Apotheosis**:
- works as described above

(4) If you use Cleansing Light **with** **Apotheosis** **and** **Isha's Encouragement**:
- Your target and all other targets are cleansed **and** all cleansed targets gain a **shield** **and** a **HoT** (Ishas Encouragement ) 


The way this skill supposed to work with both tactics slotted ( both PvE &PvP) is the way it is described on (4) since you invest in 2 tactics for a skill that is only useable if a target has ailment or hex on it.
